src/Writers/LaTeXWriter.jl

```diff
@@ -43,9 +43,11 @@ const DOCUMENT_STRUCTURE = (
 )
 
 function render(doc::Documents.Document)
+    task_local_storage(:doc, doc)
     mktempdir() do path
         cd(path) do
             local file = replace("$(doc.user.sitename).tex", " ", "")
+            cp(joinpath(doc.user.root, doc.user.source, "assets"), "assets")
```

Comment:
Yes, true. It is yet another issue caused by generating the pdf in a tmp directory.